homeassistant/components/fritz/common.py

```diff
@@ -264,6 +271,15 @@ def signal_device_update(self) -> str:
         """Event specific per FRITZ!Box entry to signal updates in devices."""
         return f"{DOMAIN}-device-update-{self._unique_id}"
 
+    def _set_guest_wifi_password(
+        self, password: str = "", length: int = DEFAULT_GUEST_WIFI_PW_LENGTH
```

Comment:
```suggestion
        self, password: str, length: int
```

You already specify the params in the call, no need to load a default